sphinx/writers/html.py

```diff
@@ -816,10 +817,17 @@ def depart_abbreviation(self, node):
     def visit_manpage(self, node):
         # type: (nodes.Node) -> None
         self.visit_literal_emphasis(node)
+        if self.manpages_url_pattern:
+            manpage = ' '.join([str(x) for x in node.children
+                                if isinstance(x, nodes.Text)])
+            node['refuri'] = self.manpages_url_pattern % manpage
+            self.visit_reference(node)
```

Comment:
Could you tell me the result if `manpage_url_pattern` enabled?
It seems this visitor method call both `visit_literal_emphasis` and `visit_reference`. So I can't guess it.

Reply:
This:

```
:manpage:`hdparm(8)`
```

... gets turned in:

```
<em class="manpage"><a class="manpage reference external" href="https://manpages.debian.org/hdparm(8)">hdparm(8)</a></em>
```

i figured i would keep the existing emphasis style.